plugins/rplanners/piecewisepolynomials/CMakeLists.txt

```diff
@@ -3,8 +3,7 @@ message(STATUS "Eigen3 version: ${Eigen3_VERSION}")
 include_directories(${EIGEN3_INCLUDE_DIRS})
 
 add_library(piecewisepolynomials SHARED polynomialcommon.h polynomialtrajectory.h polynomialtrajectory.cpp polynomialchecker.h polynomialchecker.cpp interpolatorbase.h cubicinterpolator.h cubicinterpolator.cpp quinticinterpolator.h quinticinterpolator.cpp feasibilitychecker.h generalrecursiveinterpolator.h generalrecursiveinterpolator.cpp)
-target_link_libraries(piecewisepolynomials libopenrave)
-target_link_libraries(piecewisepolynomials PRIVATE boost_assertion_failed)
+target_link_libraries(piecewisepolynomials PRIVATE boost_assertion_failed PUBLIC libopenrave)
```

Comment:
@undisputed-seraphim in some cases you have PUBLIC libraries declared first, and in others you have PRIVATE first. please be consistent.

Reply:
Noted.

Reply:
Done.